### DISCUSSION TITLE
R-Column

### DISCUSSION THREAD

**Discussion by strwberry-smggls** [elyukai/optolith-client#1591] [Q&A]
Hi, 

sorry if I simply overlooked this, but what does the "R" column on the Character Sheet mean? The "+" and "!" values have caused some major confusion in my group.

See attached picture:
<img width="822" alt="Screenshot 2022-12-28 at 15 31 05" src="https://user-images.githubusercontent.com/33194420/209827646-6427017b-636b-4157-be2e-5bede87f4f1f.png">

**Comment by elyukai** [comment]
The “R” is short for “Routine”. The column indicated if you can do a routine check, and if so, from which penalty on, based on the routine check rules. The “!” indicates that at least on of the check attributes is not on 13 and you can only do a check if you use the optional rule for routine checks, which has an influence on the minimum/maximum penalty for routine checks, which is taken into account for the displayed value.

I hope this helps! 🤗
